src/edu/wright/cs/raiderplanner/model/Exam.java

```diff
@@ -45,26 +45,51 @@ protected void replace(VersionControlEntity receivedVce) {
 		super.replace(receivedVce);
 	}
 
-	// public methods
-
-	// getters
+	/**
+	 * Get's the Resit value.
```

Comment:
`Get's` should be `Gets`. Please remove the apostrophe, as well as the other like it in this file.